src/qt/bitcoingui.cpp

```diff
@@ -364,6 +365,9 @@ void BitcoinGUI::createActions()
         connect(openAction, SIGNAL(triggered()), this, SLOT(openClicked()));
     }
 #endif // ENABLE_WALLET
+
+    new QShortcut(QKeySequence(Qt::CTRL + Qt::SHIFT + Qt::Key_C), this, SLOT(showDebugWindowActivateConsole()));
```

Comment:
Right. It looks a bit strange. But i think it should compile fine and leaves no men-leak (and requires less code).